src/app/convai-checker.component.spec.ts

```diff
@@ -275,7 +275,9 @@ function getCopyOfDefaultDemoSettings(): DemoSettings {
     hideLoadingIconAfterLoad: DEFAULT_DEMO_SETTINGS.hideLoadingIconAfterLoad,
     hideLoadingIconForScoresBelowMinThreshold:
       DEFAULT_DEMO_SETTINGS.hideLoadingIconForScoresBelowMinThreshold,
-    userFeedbackPromptText: DEFAULT_DEMO_SETTINGS.userFeedbackPromptText
+    userFeedbackPromptText: DEFAULT_DEMO_SETTINGS.userFeedbackPromptText,
+    alwaysHideLoadingIcon: DEFAULT_DEMO_SETTINGS.alwaysHideLoadingIcon,
+    loadingIconStyle: DEFAULT_DEMO_SETTINGS.loadingIconStyle
```

Comment:
I think this way of copying the settings is a little clunky and error prone, better to use 
JSON.parse(JSON.stringify(obj)) 
to get a deep copy of a trivial object.

Reply:
Done